app/views/layouts/application.html.erb

```diff
@@ -14,9 +14,9 @@
 <body>
   <%= render 'shared/navbar' %>
   <%= render 'shared/flashes' %>
-  <div class="container p-5 mt-3", style="background-color: rgba(255, 255, 255, 0.9);">
```

Comment:
Removing this from this view means you need to place it in other views, otherwise every page no longer has a container.